content/rc/subscriptions/create-fixed-subscription/fixed-plan-details.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: Fixed plan details
+description: Provides detailed information on Fixed plans
+weight: $weight
+alwaysopen: false
+categories: ["RC"]
+---
+
+## Current plans
+
+These plans are currently offered for all new and upgraded Fixed subscriptions as of February 2024.
```

Comment:
DO we want to say February?

Reply:
This date is when the table was last updated. I could make that clearer by saying the following: 
```suggestion
These plans are currently offered for all new and upgraded Fixed subscriptions (updated February 2024).
```